sdk/avalon_sdk/ethereum/ethereum_listener.py

```diff
@@ -220,25 +229,53 @@ def __init__(self, config):
             callback(normalized_event, *kargs, **kwargs)
             self.queue.task_done()
 
-    async def start(self, eventListener, callback, *kargs, **kwargs):
+    async def sync_handler(self, *kargs, **kwargs):
```

Comment:
The function name sync_handler() for an async function type looks like an oxymoron :-). I guess we could just name it like normalize_event_handler().